CUDADataFormats/SiPixelCluster/interface/gpuClusteringConstants.h

```diff
@@ -13,9 +13,9 @@ namespace gpuClustering {
   // tested on MC events with 55-75 pileup events
   constexpr uint32_t maxHitsInIter() { return 160; }  //TODO better tuning for PU 140-200
 #endif
-  constexpr uint32_t maxHitsInModule() { return 1024; }
+  constexpr uint32_t maxHitsInModule() { return 2048; }
 
-  constexpr uint32_t maxNumDigis = 3 * 256 * 1024;  // @PU=200 µ=530k σ=50k this is >4σ away
+  constexpr uint32_t maxNumDigis = 3 * 256 * 1024;  // @PU=200 µ=530 sigma=50k this is >4sigma away
```

Comment:
why the change from `σ` to `sigma` ?

Reply:
I think it was not properly rendered in my IDE but now it is. I'll revert this.